docs/QUICKSTART.md

```diff
@@ -0,0 +1,157 @@
+## Introduction
+
+## Integration
+
+*build.gradle*
+```
+implementation(org.eclipse.kuksa:kuksa-sdk:<VERSION>)
+```
+
+## Connecting to the DataBroker
+
+You can use the following snippet for a simple (unsecure) connection to the DataBroker. See the samples package for
+a detailed implementation or how to connect in a secure way with a certificate.
+
+```
+private var dataBrokerConnection: DataBrokerConnection? = null
+
+fun connectInsecure(host: String, port: Int) {
+    lifecycleScope.launch {
+        val managedChannel = ManagedChannelBuilder.forAddress(host, port)
+            .usePlaintext()
+            .build()
+
+        val connector = DataBrokerConnector(managedChannel)
+        try {
+            dataBrokerConnection = connector.connect()
+            // Connection to the DataBroker successfully established
+        } catch (e: DataBrokerException) {
+            // Connection to the DataBroker failed
+        }
+    }
+}
+```
+
+## Interacting with the DataBroker
+
+```
+fun fetch() {
+    lifecycleScope.launch {
+        val property = Property("Vehicle.Speed")
+        val response = dataBrokerConnection?.fetch(property) ?: return@launch
+        val value = response.firstValue
+        val speed = value.float
+    }
+}
+
+fun update(property: Property, datapoint: Datapoint) {
+    lifecycleScope.launch {
+        val property = Property("Vehicle.Speed")
+        val datapoint = Datapoint.newBuilder().float = 50f
+        dataBrokerConnection?.update(property, datapoint)
+    }
+}
+
+ fun subscribe(property: Property) {
+        val propertyListener = object : PropertyListener {
+            override fun onPropertyChanged(vssPath: String, field: Types.Field, updatedValue: Types.DataEntry) {
+                 when (vssPath) {
+                    "Vehicle.Speed" -> {
+                        val speed = updatedValue.value.float
+                    }
+                }
+            }
+        }
+
+        dataBrokerConnection?.subscribe(property, propertyListener)
+    }
+```
+
+## Specifications Symbol Processing
+
+The generic nature of using the `Property` API will result into an information loss of the type which can be seen in 
+the `subscribe` example. It requires an additional check of the `vssPath` when receiving an updated value to correctly 
```

Comment:
> It requires an additional check of the `vssPath`

Well, not necessarily, when I use one listener per property ;) 
Nothing to do here.